pkgs/development/tools/documentation/gi-docgen/default.nix

```diff
@@ -1,75 +1,24 @@
 { lib
 , fetchFromGitLab
-, fetchpatch
 , meson
 , ninja
 , python3
 }:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "gi-docgen";
-  version = "2021.2";
+  version = "2021.5";
 
   format = "other";
 
   src = fetchFromGitLab {
     domain = "gitlab.gnome.org";
-    owner = "ebassi";
+    owner = "GNOME";
```

Comment:
What is the reason for changing to ebassi's repo? Doesn't he make merge requests to the main GNOME repo?

Reply:
It is the other way around. It was moved from a private repo to GNOME namespace.

Reply:
Oh sorry, my bad, I read it wrongly. Then it's all good